examples/dwarfdump.rs

```diff
@@ -223,6 +227,7 @@ fn main() {
         flags.pubtypes = true;
         flags.aranges = true;
     }
+    flags.match_units = matches.opt_str("m");
```

Comment:
Create the `Regex` here, and print an error if it fails (instead of unwrap).

Reply:
Done.